test/rlx_release_SUITE.erl

```diff
@@ -906,11 +874,9 @@ make_config_script_release(Config) ->
     rlx_test_utils:create_app(LibDir1, "non_goal_1", "0.0.1", [stdlib,kernel], [lib_dep_1]),
     rlx_test_utils:create_app(LibDir1, "non_goal_2", "0.0.1", [stdlib,kernel], []),
 
-    ConfigFile = filename:join([LibDir1, "relx.config"]),
-    rlx_test_utils:write_config(ConfigFile,
-                 [{release, {foo, "0.0.1"},
-                   [goal_app_1,
-                    goal_app_2]}]),
+    %% RelxConfig = [{release, {foo, "0.0.1"},
+    %%                [goal_app_1,
+    %%                 goal_app_2]}],
```

Comment:
what's up with these being commented? Not an equivalent test case?

Reply:
This is just a test I haven't got to yet and wanted to get rid of compile warnings since `RelxConfig` isn't used yet. I did a global replace of the writing of the config file to disk which resulted in each test case now creating that variable, but still need to be updated to use it.

Reply:
Fixed.